### DISCUSSION TITLE
Relationship with Pinia & Addressing Confusion

### DISCUSSION THREAD

**Discussion by wobsoriano** [posva/pinia-colada#31] [Questions] [answered]
Hey! I posted this project on [r/vuejs](https://www.reddit.com/r/vuejs/comments/1c80l4h/pinia_colada_wip_the_smart_data_fetching_layer/) and got some questions there:

1. There's a project named [Colada](https://colada.dev/), which enables time-travel debugging for Pinia and might probably confuse some. Any thoughts on this?
2. How is it in any way related to pinia?

Thank you!

**Comment by posva** [comment, marked as answer]
Hello! Thanks for the interest and sharing it on Reddit.

1. The name might be similar but it’s different (1 vs 2 words), so I think it’s fine. When I created pinia, I was aware of the potential for something named pinia colada, I have been wanting to build something with that name for a long time 😁
2. It’s a Vue plugin that relies on Pinia stores. It needs many of the stores’ features to work

**Reply by posva** [reply]
BTW, I will definitely add longer explanations to the why section of the docs

**Reply by wobsoriano** [reply]
Thank you for answering @posva! I'll be ff. this project and use it in the future :)

**Reply by zambotn** [reply]
While i see the point of "1 vs 2 words", **the package** is called just "@pinia/colada" which IS confusing. Maybe it's just me but i find it confusing. Following your "1 vs 2 words" should be "@pinia/piniacolada"... but i would also understand "@pinia/colada-datafetch" or any additional indication to discern it from the pre-existing Colada

TBH i think that if you _really_ wanted to build something with that name you should have done it earlier, like you do with WIP PR, or at you should have least kindly asked to reserve that name for you in the pinia README... that said is already amazing we can have opensource libraries like that: if lawyers are not involved naming issues are not so important.
